scanpipe/pipes/d2d.py

```diff
@@ -664,14 +673,19 @@ def map_about_files(project, logger=None):
     for about_file_resource in from_about_files:
         _map_about_file_resource(project, about_file_resource, to_resources)
 
+        about_file_companions = about_file_resource.siblings().filter(
```

Comment:
Shouldn't we limit the QS to specific ABOUT files known extensions?

It seems that there's a risk of selecting unwanted files.

Reply:
Yes, added another filter with known extensions. Thanks!